lib/cucumber/cli.js

```diff
@@ -42,6 +43,10 @@ function Cli(argv) {
       var fullArgs = argv.slice(0, 2).concat(profileArgs).concat(argv.slice(2));
       program = getProgram();
       program.parse(fullArgs);
+    } 
+    if (program.init) { 
+      Cucumber.Cli.ProjectInitializer.createDefaultProject();
+      process.exit(0);
```

Comment:
I don't like this code being in the getConfiguration function. Since its just a getter, it really shouldn't have side effects. I'd rather it be in the `run` function. 

Also I'd prefer to not have this code actually call `process.exit`. I'm generally against any class touching globals expect at the highest level (`bin/cucumber.js`). It just makes things easier to test and to me easier to understand. If we put this in the `run` function, we can just execute the callback with success and the caller will exit with the correct value.